src/tigerbeetle/cli.zig

```diff
@@ -133,330 +164,152 @@ pub const Command = union(enum) {
 /// Parse the command line arguments passed to the `tigerbeetle` binary.
 /// Exits the program with a non-zero exit code if an error is found.
 pub fn parse_args(allocator: std.mem.Allocator) !Command {
-    var path: ?[:0]const u8 = null;
-    var cluster: ?[]const u8 = null;
-    var replica: ?[]const u8 = null;
-    var replica_count: ?[]const u8 = null;
-    var addresses: ?[]const u8 = null;
-    var cache_accounts: ?[]const u8 = null;
-    var cache_transfers: ?[]const u8 = null;
-    var cache_transfers_posted: ?[]const u8 = null;
-    var storage_size_limit: ?[]const u8 = null;
-    var cache_grid: ?[]const u8 = null;
-    var verbose: ?bool = null;
-
     // This iterator owns the arguments and is passed to the caller as a part of `Command`.
     var args = try std.process.argsWithAllocator(allocator);
     errdefer args.deinit(allocator);
 
     // Skip argv[0] which is the name of this executable
-    const did_skip = args.skip();
-    assert(did_skip);
+    assert(args.skip());
```

Comment:
I suggest `assert(args.skip())` as the canonicall way to skip the argument, given that we require ReleaseSafe (that is, I am ok with simple side-effects in assertions).